src/components/Navbar/index.jsx

```diff
@@ -89,7 +89,7 @@ function Navbar({ onClick, open, setOpen, user, setUser, setLogout }) {
                       className="nav-menu-item"
                       onClick={() => navigate("/dummygram/favourites")}
                     >
-                      <Typography fontFamily="serif" fontSize="1rem">
+                      <Typography fontFamily="Poppins" fontSize="1rem">
```

Comment:
removing the attribute fontFamily should've worked as it is inherited from the root/index... If it doesn't, means some other component with other property exists in between and needs to be fixed as well. Can you please check and confirm?

Reply:
Yeah sure, I will check this. @narayan954

Reply:
Hello @narayan954 . Yeah you're right, removing the attribute fontFamily worked and solved this issue. Should I commit the changes?